CHANGELOG.md

```diff
@@ -1,6 +1,13 @@
 # Changelog
 
 ## Unreleased
+## v0.69.0
+
+Updates smartagent, javaagent, and opentelemetry to `v5.27.2`, `1.20.0`, and `v0.69.0`/`v0.69.1` respectively.
```

Comment:
mind using the existing boilerplate for otelcol? The other items could be individual statements like we've done in the past as a clear convention.